### ISSUE TITLE
Setting I2C address on temperature sensors does not work

### ISSUE THREAD

**Issue by mwittig** [mwittig/pimatic-johnny-five#47]
From https://forum.pimatic.org/topic/2416/2nd-bmp180-on-pi/5

I've received the BMP280 and connected to the Pi IO.
So the bus looks good. I see all devices on the Bus.
I can create the second the device w/o any issues, but the measurement value is the same as the first one. The sensors are in different areas one is in a pressure box and one measuring the "normal" pressure. When is increase the pressure in "Druckdose" than Umgebungsdruck increases as well. 
I've a attached a part of my config.
```
{
      "id": "Druck",
      "name": "Druckdose",
      "class": "JohnnyFiveTemperaturePressure",
      "boardId": "1",
      "controller": "BMP180",
      "interval": 1800,
      "units": "metric",
      "xAttributeOptions": [],
      "address": "0x77"
    },
    {
      "id": "Umgebungsdruck",
      "name": "Umgebungsdruck",
      "class": "JohnnyFiveTemperaturePressure",
      "boardId": "1",
      "interval": 1800,
      "units": "metric",
      "xAttributeOptions": [],
      "address": "0x76",
      "controller": "BMP180"
    }
```
Looks like that the address attribute is ignored somewhere?!

**Comment by mwittig** [comment]
released pimatic-johnny-five@0.9.6

**Comment by mwittig** [comment]
Fix confirmed
